iocore/net/SSLUtils.cc

```diff
@@ -2159,7 +2123,7 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
   ssl_ticket_key_block *keyblock = NULL;
   if (cc == NULL || cc->keyblock == NULL) {
     // Try the default
-    keyblock = global_default_keyblock;
+    keyblock = params->default_global_keyblock;
```

Comment:
OK, so as long as ``ssl_callback_session_ticket`` is called just once before the SSL config is destroyed we are OK. I'd feel more comfortable if there was a way to remove the keyblock from the SSL context after use so that we have less chance of a dangling pointer.